webpack.config.js

```diff
@@ -75,6 +75,30 @@ const babelLoader = {
 	},
 };
 
+/**
+ * Converts @wordpress require into window reference
+ *
+ * Note this isn't the same as camelCase becuase of the
```

Comment:
Typo here: `becuase` should be `because`. Also, perhaps we want to change `camelCase` to `_.camelCase` to be more specific.

Reply:
thanks for the typo. interestedly though I wasn't referring to `_.camelCase`. maybe I should write `camel case` instead? I wanted to refer to the algorithm, not the lodash or underscore implementation…

Reply:
`camel case` should be even better as it's more broad indeed 👍

Reply:
space added!